dockerfiles/rhel.entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2021 Red Hat, Inc.
```

Comment:
new file, new copyright date. I know this is boilerplate, but please set a valid start date of 2021.